src/include/dhandle.h

```diff
@@ -112,4 +112,18 @@ struct __wt_data_handle {
 #define WT_DHANDLE_OPEN 0x80u         /* Handle is open */
                                       /* AUTOMATIC FLAG VALUE GENERATION STOP */
     uint32_t flags;
+
+/* AUTOMATIC FLAG VALUE GENERATION START */
+#define WT_DHANDLE_ASSERT_TS_READ_ALWAYS 0x001u      /* Assert read always checking. */
+#define WT_DHANDLE_ASSERT_TS_READ_NEVER 0x002u       /* Assert read never checking. */
+#define WT_DHANDLE_ASSERT_TS_WRITE_ALL 0x004u        /* Assert write all checking. */
+#define WT_DHANDLE_ASSERT_TS_WRITE_DIAGNOSTIC 0x008u /* Assert write diagnostic. */
+#define WT_DHANDLE_TS_ALWAYS 0x010u                  /* Handle using always checking. */
+#define WT_DHANDLE_TS_KEY_CONSISTENT 0x020u          /* Handle using key consistency checking. */
+#define WT_DHANDLE_TS_MIXED_MODE 0x040u /* Handle using mixed mode timestamps checking. */
+#define WT_DHANDLE_TS_NEVER 0x080u      /* Handle not using timestamps checking. */
```

Comment:
That is not what `NEVER` means. It means check that this handle is never using timestamps.

Reply:
Addressed [here](https://github.com/wiredtiger/wiredtiger/pull/6121#discussion_r538405871).